public/blog/how-were-making-installation-issues-a-thing-of-the-past.md

```diff
@@ -63,7 +63,7 @@ You'll notice that the beta version installs almost instantly, and no compilatio
 
 As we gain responses from the community about this new version we'll likely make changes to ensure we've sidestepped all the installation issues that have plagued users in the past. Once that's finished we'll make an official release. As well, we'll be performing this same work for the TestRPC in the coming weeks to ensure the TestRPC has the same hassle-free, instant installation that you'll soon come to expect from Truffle.
 
-If you run into issues with the beta version, don't hesitate to [write up an issue](https://github.com/trufflesuite/truffle/issues) in our issue tracker or reach out to us on [Gitter](https://gitter.im/ConsenSys/truffle). We're happy to help.
+If you run into issues with the beta version, don't hesitate to [write up an issue](https://github.com/trufflesuite/truffle/issues) in our issue tracker or reach out to fellow Trufflers for help from the community on [Gitter](https://gitter.im/ConsenSys/truffle). We're happy to help.
```

Comment:
Same as above - `community` feels redundant. I'd change to:

> reach out to your fellow Trufflers for help on Gitter

Also, if people are reaching out to our fellow Trufflers, you should probably remove or rewrite the next line, `We're happy to help` so that you remove the `we`.